source/features/clone-branch.tsx

```diff
@@ -71,27 +73,29 @@ async function cloneBranch({delegateTarget: cloneButton}: delegate.Event<MouseEv
 
 async function init(): Promise<void | false> {
 	await api.expectToken();
-	const deleteIcons = select.all([
+	const deleteIconClass = [
 		'branch-filter-item-controller .octicon-trashcan', // Pre "Repository refresh" layout
 		'branch-filter-item .octicon-trashcan'
-	]);
+	].join();
 	// If the user does not have rights to delete a branch, they can’t create one either
-	if (deleteIcons.length === 0) {
+	if (!await elementReady(deleteIconClass)) {
```

Comment:
We probably don’t need both elementReady and observe. Let’s simplify

Reply:
I did it for the return false. Drop it?

Reply:
> I did it for the return false. Drop it?

Yeah.

`return false` generally nice to have, but not worth it if we have to keep otherwise-completely-useless code.